sources/packages/web/src/views/institution/DynamicInstitutionProfile.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <Message severity="info">
```

Comment:
Can we move this warning message to the formio definition?
I think that this would be one thing that would be good to be dynamic.
If the student profile is not dynamic, it would be good to change there also in the future.

Reply:
I think form.io should only manage form rendering and handling events. If require, we can develop another reusable view with accessory view, then singular responsibility principle will be maintained.

Reply:
I don't think that the singular responsibility principle applies to this here.
The message is on the top just because we as developers decided to put it there. It belongs to the form context and it maybe would be rendered right in the middle of the form in the future. This is probably where the ministry could have some benefits of doing changes without code changes.
Create a new formio container only to hold the message seems that we would be creating a "static" place where the message would be displayed.

Reply:
But message is only attached with profile forms only, even no use for application forms.

Reply:
I will create one technical debt to handle this change for both of the profile (student, institution) info.